docs/scenarios/library-authoring/trimming.md

```diff
@@ -0,0 +1,177 @@
+# How to trim your package dependencies
+
+By [Phillip Carter](https://github.com/cartermp)
+
+This article covers what you need to know about "trimming" your package dependencies to use only the packages you need for .NET Core RC2.
+
+## What trimming means
+
+.NET Core RC2 introduces the .NET Standard Library metapackage, which is a NuGet package composed other packages.  It is curated to be appropriate for developing libraries.  However, there's a good chance that your library won't use every single package it contains.  When authoring a library and distributing it over NuGet, it's a best practices to "trim" your dependencies down to only the packages you actually use.  This results in a smaller overall footprint for NuGet packages.
+
+## How to trim
+
+Currently, there is no official tooling which supports the trimming of packages.  You'll have to do it manually.  The general process will look like this:
```

Comment:
the trimming of packages -> package trimming?
will look like this -> looks like the following